_posts/documentation/explore/2000-01-07-related-projects.md

```diff
@@ -67,6 +68,8 @@ Several page capture projects utilizing PhantomJS:
 
  * [Compass Magick](https://github.com/StanAngeloff/compass-magick) is a dynamic image generation for Compass using (among others) PhantomJS.
 
+ * [DeckTape](https://github.com/astefanutti/decktape) is a high-quality PDF exporter from HTML presentation frameworks like impress.js and reveal.js
```

Comment:
DeckTape provides a CLI to get screenshots from HTML presentations, similarly to _pageres_, hence the item added in the _Screenshots Utilities_ section. It provides in addition a CLI to export HTML presentations to PDF documents, similarly to _Slippy_, hence the item added in the _Tools_ section.

As I noted that the _pageres_ entry is duplicated to make this kind of distinction, I went for adding two entries. That being said, I would understand that you consider the distinction not worth the duplication.